lib/Markup/LineList.cpp

```diff
@@ -43,20 +43,11 @@ std::string LineList::str() const {
 }
 
 size_t swift::markup::measureIndentation(StringRef Text) {
-  size_t Col = 0;
-  for (size_t i = 0, e = Text.size(); i != e; ++i) {
-    if (Text[i] == ' ' || Text[i] == '\v' || Text[i] == '\f') {
-      ++Col;
-      continue;
-    }
-
-    if (Text[i] == '\t') {
-      Col += ((i + 8) / 8) * 8;
-      continue;
-    }
-    return i;
-  }
-  return Text.size();
+  static constexpr llvm::StringLiteral IndentChars(" \v\f\t");
+  size_t FirstNonIndentPos = Text.find_first_not_of(IndentChars);
+  if (FirstNonIndentPos == StringRef::npos)
+    return Text.size();
+  return FirstNonIndentPos;
```

Comment:
I was about to say this isn't NFC because of the tabstop, but then I realized `Col` isn't actually being returned, which is what the warning is about in the first place. Given this is used to eg. trim off indentation, it doesn't seem like we want this to return `Col` though so... 🤷 In any case, it's NFC as is.

Reply:
Yeah... I'm not sure why it was written that way but it was done in 2015, not sure if anyone would remember exactly why it happened that way. Thanks for the review!